sdk/src/driftClient.ts

```diff
@@ -360,6 +360,10 @@ export class DriftClient {
 				this.userStats.unsubscribe()
 			);
 		}
+		if (this.userAccountSubscriptionConfig.type === 'polling') {
+			this.userAccountSubscriptionConfig.accountLoader.stopPolling();
```

Comment:
Not kosher because assumption is that single bulk loader is used for all polling. DriftClient unsubscription shouldn't cause global unsubscription

Reply:
The only thing this unsubscribe method should do is remove any DriftClient polling from the account loader. I believe that's what the lines of code directly above this change are already doing (or trying to do, if they're not removing everything, then that is the _true change_ that needs to happen)